lib/guest_installation_and_configuration_base.pm

```diff
@@ -772,7 +772,12 @@ sub config_guest_network_bridge_device {
         if ($self->{guest_netaddr} ne 'host-default') {
             $self->write_guest_network_bridge_device_config($_bridge_network, $_bridge_device, 'static', 'auto', '', 'yes');
             my $_bridge_device_config_file = '/etc/sysconfig/network/ifcfg-' . $_bridge_device;
-            script_retry("wicked ifup $_bridge_device_config_file $_bridge_device", retry => 3, die => 0);
+            script_retry("wicked ifup $_bridge_device_config_file $_bridge_device", retry => 3, die => 0) if is_sle;
+            #Not sure if it is a bug, but it works in openSUSE this way.
+            if (is_opensuse) {
```

Comment:
Please do not use "is_opensue" or "is_sle" here. Please use if ($self->${guest_os_name} eq 'xxx' or ne 'yyy')